tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
+	"compilerOptions": {
+		"allowJs": true,
+		"allowSyntheticDefaultImports": true,
+		"checkJs": true,
+		"jsx": "preserve",
+		"lib": ["dom", "esnext"],
+		"module": "commonjs",
+		"noEmit": true,
+		"resolveJsonModule": true,
+		"target": "esnext",
+
+		/* Strict Type-Checking Options */
+		"strict": true,                        /* Enable all strict type-checking options. */
+		"noImplicitAny": false,                /* Raise error on expressions and declarations with an implied 'any' type. */
+
+		/* Additional Checks */
+		"noUnusedLocals": true,                /* Report errors on unused locals. */
+		"noUnusedParameters": true,            /* Report errors on unused parameters. */
+		"noImplicitReturns": true,             /* Report error when not all code paths in function return a value. */
+		"noFallthroughCasesInSwitch": true,    /* Report errors for fallthrough cases in switch statement. */
+
+	},
+	"include": [
```

Comment:
The `include` array will be changed with each new package that is added. We're skipping tests because almost universally throughout the project, incorrect partial types are fed to functions to test a specific case which is not allowed as far as TypeScript is concerned. Fixing all of those instances would require a pretty significant amount of work and there's no real benefit to doing it.

Reply:
Yes, let's skip tests for now and revisit our approach once we have all packages enabled here 👍